packages/ra-ui-materialui/src/input/NullableBooleanInput.tsx

```diff
@@ -30,7 +30,12 @@ const getStringFromBoolean = (value?: boolean | null): string => {
 };
 
 const NullableBooleanInput: FunctionComponent<
-    InputProps<TextFieldProps> & Omit<TextFieldProps, 'label' | 'helperText'>
+    InputProps<TextFieldProps> &
```

Comment:
this type is becoming increasingly complex. Anyone trying to extend `NullableBooleanInput` in TypeScript will fight trying to type its props. 

Please create a custom `NullableBooleanInputProps` interface and export it.